source/tools/kubernetes/dask-operator.md

```diff
@@ -1 +1,381 @@
+              nvidia.com/gpu: "1"
+  scheduler:
+    # ...
+```
+
+Inside our pod spec we are configuring one container that uses the `rapidsai/rapidsai-core` container image.
+It also sets the `args` to start the `dask-cuda-worker` and configures one NVIDIA GPU.
+
+```{note}
+We also have to set the environment variable `DISABLE_JUPYTER=true` because the RAPIDS container images will run Jupyter instead of our supplied command.
+```
+
+#### Scheduler
+
+Next we have a `scheduler` section that also contains a `spec` for the scheduler pod and a `service` which will be used by the operator to create a `Service` resource to expose the scheduler.
+
+```yaml
+# ...
+spec:
+  worker:
+    # ...
+  scheduler:
+    spec:
+      containers:
+        - name: scheduler
+          image: "rapidsai/rapidsai-core:22.06-cuda11.5-runtime-ubuntu20.04-py3.9"
+          imagePullPolicy: "IfNotPresent"
+          env:
+            - name: DISABLE_JUPYTER
+              value: "true"
+          args:
+            - dask-scheduler
+          ports:
+            - name: tcp-comm
+              containerPort: 8786
+              protocol: TCP
+            - name: http-dashboard
+              containerPort: 8787
+              protocol: TCP
+          readinessProbe:
+            httpGet:
+              port: http-dashboard
+              path: /health
+            initialDelaySeconds: 5
+            periodSeconds: 10
+          livenessProbe:
+            httpGet:
+              port: http-dashboard
+              path: /health
+            initialDelaySeconds: 15
+            periodSeconds: 20
+    service:
+      # ...
+```
+
+For the scheduler pod we are also setting the `rapidsai/rapidsai-core` container image, mainly to ensure our Dask versions match between
+the scheduler and workers. We also disable Jupyter and ensure that the `dask-scheduler` command is configured.
+
+Then we configure both the Dask communication port on `8786` and the Dask dashboard on `8787` and add some probes so that Kubernetes can monitor
+the health of the scheduler.
+
+```{note}
+The ports must have the `tcp-` and `http-` prefixes if your Kubernetes cluster uses [Istio](https://istio.io/) to ensure the [Envoy proxy](https://www.envoyproxy.io/) doesn't mangle the traffic.
+```
+
+Then we configure the `Service`.
+
+```yaml
+# ...
+spec:
+  worker:
+    # ...
+  scheduler:
+    spec:
+      # ...
+    service:
+      type: ClusterIP
+      selector:
+        dask.org/cluster-name: rapids-dask-cluster
+        dask.org/component: scheduler
+      ports:
+        - name: tcp-comm
+          protocol: TCP
+          port: 8786
+          targetPort: "tcp-comm"
+        - name: http-dashboard
+          protocol: TCP
+          port: 8787
+          targetPort: "http-dashboard"
+```
+
+This example shows using a `ClusterIP` service which will not expose the Dask cluster outside of Kubernetes. If you prefer you could set this to
+`LoadBalancer` or `NodePort` to make this externally accessible.
```

Comment:
```suggestion
[`LoadBalancer`](https://kubernetes.io/docs/concepts/services-networking/service/#loadbalancer) or [`NodePort`](https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport) to make this externally accessible.
```

Since you have the reference below too